sections/04_study.md

```diff
@@ -138,7 +141,7 @@ instructive in that clever sources of data, not just more descriptive models,
 are still critical.
 
 We already understand how mis-splicing of a single gene can cause diseases such
```

Comment:
Maybe "lead to" instead of "cause"? I think it softens up the causality a little bit -- also I'm definitely not an expert on limb girdle muscular dystrophy but it's probably safe to assume that there are some cases not caused by splicing.